search/search_repository.go

```diff
@@ -685,6 +697,31 @@ func (r *GormSearchRepository) listItemsFromDB(ctx context.Context, criteria cri
 		return nil, 0, errors.NewBadParameterError("expression", criteria)
 	}
 
+	if childIterations {
+		itrRepo := iteration.NewIterationRepository(r.db)
+
+		allitrs := []string{}
+		for _, itr := range iterations {
+			childItrs, err := itrRepo.LoadChildren(ctx, itr)
+			if err != nil {
+				// Since this is a search, missing iteration is not really an error.
```

Comment:
I'm not sure I agree with this. IMHO you should error out if an iteration is not found. @michaelkleinhenz @jarifibrahim what do you think?

Reply:
@kwk I spoke to @baijum about this.
Usually, a search doesn't return an error. It just returns the data (empty if nothing is found).
We shouldn't return an error in this case.
If the space id is invalid we don't return an error, why return an error specifically for the iterations?

Reply:
Fine by me